ci/azure-test-stable.yml

```diff
@@ -20,6 +20,11 @@ jobs:
       # rust_version: stable
       rust_version: ${{ parameters.rust }}
 
+  - script: cargo install cargo-hack
```

Comment:
This is nitpicky, but it might be helpful to have a comment here explaining what we're using `cargo-hack` for and why?

Reply:
I wrote descriptions for 'check' scripts, but if necessary, I'd happy to add a description to the 'install' script.
https://github.com/tokio-rs/tokio/blob/02f7264008e0333d7c4df183d3bcfbda8a6d930f/ci/azure-test-stable.yml#L47-L50
https://github.com/tokio-rs/tokio/blob/02f7264008e0333d7c4df183d3bcfbda8a6d930f/ci/azure-test-stable.yml#L78-L81